user-guide/Advanced_Modules/DOM/DOM_objects/DomInstance.md

```diff
@@ -46,6 +46,7 @@ When something goes wrong during the CRUD actions, the `TraceData` can contain o
 | DomInstanceDoesNotContainRequiredModuleSections | The `DomInstance` does not contain exactly one `Section` for each required `SectionDefinition` defined for this module. Available properties: *DomInstance*, *MissingSections*, *InvalidSections* |
 | MultipleSectionsNotAllowedForSectionDefinition | The `DomInstance` contains more than one `Section` for a `SectionDefinition` that does not allow multiple `Sections` according to either the `SectionDefinitionLink` or the `DomStatusSectionDefinitionLink`. Available properties: *InvalidSections* |
 | ValueForSoftDeletedFieldNotAllowed | Attempted to create or update a `DomInstance` with one or multiple field values that are part of a soft-deleted `FieldDescriptor`, `SectionDefinitionLink`, or `StatusSectionDefinitionLink`, or the `DomInstance` has a `GenericEnumFieldDescriptor` with a soft-deleted entry as value. Available properties: *DomInstanceId*, *AssociatedFields*. |
+| ValueTooLarge | Attempted to create or update a `DomInstance` with a string value that is too large. The *AssociatedFields* collection contains the IDs of the `SectionDefinition`, `Section` and `FieldDescriptor` that point to the incorrect `FieldValue`, in addition to the *ActualSize* property that contains the actual UTF-8 size of the received string. This error message is given from DataMiner 10.4.12/10.5.0 onwards.<!-- RN 39496 --> |
```

Comment:
Maybe add the actual limit here as well, so people who search for this error know what the limit is?